webdriver/tests/bidi/log/entry_added/event_buffer.py

```diff
@@ -95,11 +95,11 @@ async def on_event(method, data):
     remove_listener = bidi_session.add_event_listener("log.entryAdded", on_event)
 
     # Log a message, refresh, log another message and subscribe
-    await create_log(bidi_session, top_context, log_type, "missed_message")
+    expected_text_1 = await create_log(bidi_session, top_context, log_type, "cached_message_1")
```

Comment:
In many tests, we start with a "refresh" specifically to cleanup the buffer (see L83/84 here). All those comments are actually incorrect, but I'm also wondering if this will not require further changes to the tests. Maybe we should use new tabs for each and every log test, this would avoid leaking log events I guess

Reply:
If that would resolve the issue I would also be in favor of it. We have good experience with that kind of handling from navigation tests already and it would perfectly apply here.

Reply:
We probably need both. 

If the implementation keeps events across refresh (which as @sadym-chromium points out, is expected by the spec) then it means the refresh we do at the beginning of every test to avoid "cached messages" are actually useless. The only way to avoid messages leaking across tests should be to use new tabs. 

The update proposed in the PR is still necessary, because if a refresh does not clear the buffer, this assert was wrong in the first place.

Reply:
I switched tests to the new tab.

Reply:
@whimboo @juliandescottes PTAL